Automattic/vip-go-mu-plugins
# Hotfix: HTTP Concat for Gutenberg

**The Why**

HTTP Concat plugin currently breaks Gutenberg Admin. This is a hot fix that will selectively disable HTTP Concat in the admin when editing posts that have Gutenberg enabled.

This change will not affect people that have defined `VIP_GO_DISABLE_RAMP` as true.

**Testing**
1. Activate Gutenberg on a fresh VIP GO Installation
2. Add this to the active theme functions.php:
```
wpcom_vip_load_gutenberg( [ 'post_types' => [ 'page' ] ] );
```
3. Reload the Page edit page twice, observe a white screen of death ( JavaScript errors in console )
4. Apply this patch
5. Problem fixed - Gutenberg should now work in "Add new Page".
6. Visit "Add new post" - observe concatenated scripts because Gutenberg is only enabled in "page" post type.
